packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -382,7 +396,14 @@ export default class RuntimeResolverImpl implements RuntimeResolver<Owner> {
       if (modifier !== undefined) {
         let manager = getModifierManager(owner, modifier.class!);
 
-        return new CustomModifierDefinition(name, modifier, manager!, this.isInteractive);
+        assert('Expected a modifier manager to exist', manager);
```

Comment:
Needs better error messaging (render debug stack, _any_ information about the actual thing that is `modifier.class` here, etc)